compose.yaml

```
@@ -0,0 +1,25 @@
+services:
+  velodyne:
+    image: husarion/velodyne:noetic
+    container_name: velodyne
+    network_mode: host
+    tty: true
```

```suggestion
    container_name: velodyne
    network_mode: host
```
I think `tty` in this compose file is unnecessary.